pipelines/live-1/main/check-environment.yaml

```diff
@@ -1,28 +1,18 @@
 resources:
-- name: cloud-platform-environments-repo
-  type: git
-  source:
-    uri: https://github.com/ministryofjustice/cloud-platform-environments.git
-    branch: master
-    git_crypt_key: ((cloud-platform-environments-git-crypt.key))
-- name: tools-image
+- name: ruby-image
```

Comment:
Since this shows up in other parts of the config and also the UI, should we rename it to something more specific, like the actual image name?

Reply:
Absolutely right, changed name to something more distinguished.